tasks/main.yml

```
@@ -26,6 +26,7 @@
     src: vault_config.hcl.j2
     dest: "{{ vault_configuration_path }}/config.hcl"
     mode: "u=rwx,g=rwx,o=r"
+  notify: restart vault
```

I think on first deployment this won't work: The service file is created AFTER this step.
Will this silently fail or will it abort the process?

---

Handlers, which are triggered by the notify will always run  at the end of the playbook.  So the service will be linked and there will be no error at all.

---

Didn't know that yet, cool!